docs/resources/logdna_archive.md

```diff
@@ -0,0 +1,166 @@
+# Resource: `logdna_archive`
```

Comment:
is `logdna_archive` the best name for this? it's a "configuration resource" similar to the streaming stuff, so maybe `logdna_archive_config` ? ignore me if this has already been discussed elsewhere.

Reply:
This is in line with views and alerts which uses `logdna_view` and `logdna_alert`. It is also shorter and nicer to work with and it cannot be confused with anything else I believe.